templates/chatbots/settings_content.html

```diff
@@ -0,0 +1,260 @@
+<div id="settings-wrapper">
+  <div class="flex justify-end mb-6">
+    {% if edit_mode %}
+      <div class="flex gap-2">
+        <button id="save-all-button"
+                class="btn btn-success btn-sm"
+                hx-post="/a/{{ request.team.slug }}/chatbots/{{ experiment.id }}/settings/save-all/"
```

Comment:
Use `url` template tag for urls:

```suggestion
                hx-post="{% url 'chatbots:chatbots_settings_edit_mode' request.team.slug experiment.id %}"
```